src/test/java/com/github/onsdigital/JWTVerifierImplTests.java

```diff
@@ -0,0 +1,111 @@
+package com.github.onsdigital;
+
+import com.github.onsdigital.exceptions.JWTDecodeException;
+import com.github.onsdigital.exceptions.JWTTokenExpiredException;
+import com.github.onsdigital.exceptions.JWTVerificationException;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class JWTVerifierImplTests {
```

Comment:
This should be public.

Reply:
I have upgraded this to JUnit 5. JUnit 5 now recommends that test classes and methods are package private: https://junit.org/junit5/docs/snapshot/user-guide/#:~:text=It%20is%20generally,Java%20Module%20System.